yt/data_objects/construction_data_containers.py

```diff
@@ -173,6 +182,7 @@ def __init__(
         method="integrate",
         field_parameters=None,
         max_level=None,
+        moment=1,
         *,
```

Comment:
Sorry I missed that one
```suggestion
        *,
        moment=1,
```